rfcs/text/0030-linux-event-model.md

```diff
@@ -0,0 +1,1299 @@
+    action: 'fork',
+  },
+  host: {
+    architecture: 'x86_64',
+    hostname: 'james-fleet-714-2',
+    id: '48c1b3f1ac5da4e0057fc9f60f4d1d5d',
+    ip: '127.0.0.1,::1,10.132.0.50,fe80::7d39:3147:4d9a:f809',
+    mac: '42:01:0a:84:00:32',
+    name: 'james-fleet-714-2',
+    os: {
+      Ext: {
+        variant: 'CentOS'
+      },
+      family: 'centos',
+      full: 'CentOS 7.9.2009',
+      kernel: '3.10.0-1160.31.1.el7.x86_64 #1 SMP Thu Jun 10 13:32:12 UTC 2021',
+      name: 'Linux',
+      platform: 'centos',
+      version: '7.9.2009'
+    }
+  },
+  process: {
+    id: '4321',
+    args: ['/bin/sshd'],
+    args_count: 1,
+    command_line: 'sshd',
+    executable: '/bin/sshd',
+    name: 'sshd',
+    interactive: false,
+    working_directory: '/',
+    pid: 3,
+    start: '2021-10-14T08:05:34.853Z',
+    user: {
+      id: '2',
+      name: 'kg',
+    },
+    parent: {
+      id: '4322',
+      args: ['/bin/sshd'],
+      args_count: 1,
+      command_line: 'sshd',
+      executable: '/bin/sshd',
+      name: 'sshd',
+      interactive: true,
+      working_directory: '/',
+      pid: 2,
+      start: '2021-10-14T08:05:34.853Z',
+      user: {
+        id: '0',
+        name: 'root',
+      },
+      fds: [
+        {
+          descriptor:0,
+          type: 'char_device',
+          char_device: {
+            major: 8,
+            minor: 1
+          }
+        }
+      ],
+      tty: {
+        descriptor: 0,
+        type: 'char_device',
+        char_device: {
+          major: 8,
+          minor: 1
+        }
+      }
+    },
+    group_leader: {
+      id: '4321',
+      args: ['bash'],
+      args_count: 1,
+      command_line: 'bash',
+      executable: '/bin/bash',
+      name: 'bash',
+      interactive: true,
+      working_directory: '/home/kg',
+      pid: 3,
+      start: '2021-10-14T08:05:34.853Z',
+      user: {
+        id: '2',
+        name: 'kg',
+      },
+      fds: [
+        {
+          descriptor:0,
+          type: 'char_device',
+          char_device: {
+            major: 8,
+            minor: 1
+          }
+        }
+      ],
+      tty: {
+        descriptor: 0,
+        type: 'char_device',
+        char_device: {
+          major: 8,
+          minor: 1
+        }
+      }
+    },
+    session_leader: {
+      id: '4321',
+      args: ['bash'],
+      args_count: 1,
+      command_line: 'bash',
+      executable: '/bin/bash',
+      name: 'bash',
+      interactive: true,
+      working_directory: '/home/kg',
+      pid: 3,
+      start: '2021-10-14T08:05:34.853Z',
+      user: {
+        id: '2',
+        name: 'kg',
+      },
+      fds: [
+        {
+          descriptor:0,
+          type: 'char_device',
+          char_device: {
+            major: 8,
+            minor: 1
+          }
+        }
+      ],
+      tty: {
+        descriptor: 0,
+        type: 'char_device',
+        char_device: {
+          major: 8,
+          minor: 1
+        }
+      }
+    },
+    entry_leader: {
```

Comment:
It seems like this RFC may have had changes to it since these examples were generated? The note on `process.entry_leader` says what's below but this example has a lot more details filled in. This same comment applies more broadly to these examples.

```
      - at: process.entry_leader
        as: parent 
        short_override: Information about the entry leaders parent. Only pid, start and entity_id fields are set.
```

Reply:
So, this new addition allows for the nesting of an 'anemic' parent to the entry_leader. Which will allow us to pull a thread to navigate upwards from a session to it's parent. 
e.g
process: {
   ...
   entry_leader: {
      ...
      parent: {
         pid: ...,
         start: ...
         entity_id: ...
      }
   }
}

I'll update all the examples to include these anemic processes.